CONTRIBUTING.md

```diff
@@ -13,3 +13,30 @@ the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/pub
 All contributions to this project will be released under the CC0
 dedication. By submitting a pull request, you are agreeing to comply
 with this waiver of copyright interest.
+
+## Starting Work on an Issue
+
+Issues that are marked with the `ready` label are ripe for the picking!  Simply assign yourself to the issue to and change its label to `in progress` to indicate that you are working on it.
+
+If the issue involves writing code or producing some other change that will result in a pull request, begin by creating yourself a branch with a short descriptive name of the work that includes the issue number at the end, e.g., `document-pr-process-#36` - if you are not a part of the 18F Team, please fork the repository first and then create a branch for yourself with the same convention.
```

Comment:
Do any file changes necessitate a pull request? I don't think that's something we discussed specifically at the meeting the other day. Anyway, if so, I think it might be clearer to say something like "If the issue involves updating files, you will need to create a pull request once your updates are complete. Begin by..."